partEZ/app/Http/Controllers/EventController.php

```diff
@@ -66,6 +66,92 @@ public function details($eid)
             ->with('chat_messages', $chat_messages);
     }
 
+        /**
+     * Show the detail screen for an event.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function detailsEdit($eid)
+    {
+        $event = Event::getEvent($eid);
+        $invites = Self::getInvitesFromEid($eid);
+        $all_poll_options = Self::getPollOptionsFromEid($eid);
+        $itemslist = Event::getEventItems($eid);
+        $items = [];
+
+        foreach ($itemslist as $item)
+        {
+            array_push($items, $item);
+        }
+        $chat_messages = MessageController::getMessagesFromEid($eid);
```

Comment:
I don't think we need to grab the chat log for this page, it doesn't get used at all in this function.